index.html

```diff
@@ -131,4 +137,4 @@ <h3><a href="ideas/co-design-draft-proposal">IDRC Inclusive Co-Design Draft Prop
 	</script>
 
 </body>
-</html>
+</html>
```

Comment:
Generally, you should configure your editor to leave a blank line at the end of files automatically for you.

Reply:
https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file

Reply:
https://github.com/mattst/sublime-single-trailing-new-line

?

Reply:
I'm not a Sublime user myself, but that's probably a fine plugin if the default settings don't work for you. Perhaps try them for a while first and see how it goes?

Reply:
Yep just installed it and will see!